### PR TITLE
Cypress test for issue 1750.

### PR DESCRIPTION
**Pull request by dvkruchinin**
Add Cypress test for issue [1750](https://github.com/opencv/cvat/issues/1750)

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by bsekachev** [comment]
Some conflicts appeared here

**Comment by dvkruchinin** [comment]
Rebased.

File: tests/cypress/integration/issue_1750_err_aam_switch_frames.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+    describe(`Testing issue "${issueId}"`, () => {
+        it('Create multiple objects', () => {
+            cy.createShape(309, 431, 409, 531)
+            cy.createShape(200, 300, 300, 400)
+        })
+        it('Go to AAM', () => {
+            cy.get('.cvat-annotation-header-right-group')
+            .find('.ant-select-selection-selected-value')
+            .click()
```
Comment (bsekachev) [inline tests/cypress/integration/issue_1750_err_aam_switch_frames.js:48]: Could we use here ``cvat-workspace-selector`` class?

Comment (dvkruchinin) [reply tests/cypress/integration/issue_1750_err_aam_switch_frames.js]: Yes, we can. Replaced.

Comment (bsekachev) [reply tests/cypress/integration/issue_1750_err_aam_switch_frames.js]: Great

**Pull request merged**